frontend/src/BabyPage.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import {BabyModel} from "./BabyModel";
+import {useParams} from "react-router";
+import styled from "styled-components";
+import {Link} from "react-router-dom";
+
+type babyProps = { babies: BabyModel[] }
```

Comment:
You define babies to be a list that is passed from another component. BabyPage.tsx is displaying only one baby though.

Consider passing one baby to BabyPage.tsx instead of the whole list